flow360/component/case.py

```diff
@@ -447,18 +447,6 @@ def results(self) -> CaseResults:
         """
         return self._results
 
-    def download_log(self, log_file, to_file=".", keep_folder: bool = True):
```

Comment:
will need to refactor test cases that call this function in _test_surface_mesh.py and _test_volume_mesh.py

Reply:
removed files _test_surface_mesh.py and _test_volume_mesh.py as we should not be running tests that require cloud access